packages/template/components/Avatar.jsx

```diff
@@ -0,0 +1,21 @@
+export function Avatar({ name, img, href }) {
+  const Component = href ? "a" : "div";
+  return (
+    <Component href={href} className="group block flex-shrink-0 mt-2">
+      <div className="flex items-center space-x-2">
+        <div>
+          <img
+            className="inline-block h-9 w-9 rounded-full"
+            src={img}
+            alt={name}
+          />
+        </div>
+        <div className="ml-3">
```

Comment:
let's try and use standard named colors - see https://github.com/flowershow/flowershow/issues/351